src/coreclr/jit/compiler.h

```diff
@@ -9700,7 +9694,6 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
         STRESS_MODE(MAKE_CSE)                                                                   \
         STRESS_MODE(LEGACY_INLINE)                                                              \
         STRESS_MODE(CLONE_EXPR)                                                                 \
-        STRESS_MODE(USE_CMOV)                                                                   \
```

Comment:
Removing this will affect which stress modes get enabled in which methods for jitstress. You might want to run the jit stress jobs to verify that nothing new pops up or revert this removal (I can remove it as part of #83771).

Reply:
Ok, I will revert this.